### PR TITLE
manual upgrade clarification

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

**Comment by rnovikovP** [comment]
we see https://perconacorp.slack.com/archives/C03FB1CAZH9/p1688715789544119 that in some cases people reading our RN and blog as "you can't now use UI to get updates" so looking for some clarification here, to make it clear what is limited exactly.

**Comment by rnovikovP** [comment]
I think we need to rewrite the full section about the upgrade: 
Start from :  

- People using docker upgrade - will be fine 
- Update button users  - can continue using the button but then they'll not get the base OS updated or execute manual upgrade based on their distribution: docker , ami/ovf  - links to specific instructions how to upgrade not using  upgrade button

File: docs/release-notes/2.38.0.md [hunk 1]
```diff
@@ -45,12 +45,7 @@ The following users will be able to take advantage of this immediately:
     Your existing `docker pull` or `docker run` commands will use the new image.
 
 **Manual upgrade options**
-
-If you are upgrading through the UI’s **Upgrade Now** button, you'll have to wait a little longer for an automated migration path. This includes:
-
-- Docker users that use UI upgrade
-- AMI users
-- OVF users
+AMI, OVF users and Docker users who want to upgrade PMM via the **Upggrade Now** option on the Home Dashboard will have to wait a little longer for an automated migration path from Centos 7 to Oracle Linux 9.
```
Comment (BupycHuk) [inline docs/release-notes/2.38.0.md:48]:
```suggestion
AMI, OVF users and Docker users who want to upgrade PMM via the **Upgrade Now** option on the Home Dashboard will have to wait a little longer for an automated migration path from Centos 7 to Oracle Linux 9.
```

**Review by BupycHuk — APPROVED** [review]

**Review by rnovikovP — CHANGES_REQUESTED** [review]
https://github.com/percona/pmm-doc/pull/1112#issuecomment-1661853754

**Pull request closed without merging**